webhooks/pod_webhook.go

```diff
@@ -173,6 +173,11 @@ func (m *PodMutator) injectSidecar(pod *corev1.Pod, configMap string, featureFla
 		FlagDTag = os.Getenv("FLAGD_VERSION")
 	}
 
+	var envs []corev1.EnvVar
+	if featureFlag.Spec.FlagDSpec.Envs != nil {
```

Comment:
```suggestion
	if featureFlag.Spec.FlagDSpec != nil {
```
it's the FlagDSpec that's causing the panic

Reply:
ta